OpenMarket/OpenMarket/Utils/NumberFormatter.swift

```diff
@@ -0,0 +1,25 @@
+//
+//  NumberFormatter.swift
+//  OpenMarket
+//
+//  Created by 박병호 on 2022/01/13.
+//
+
+import Foundation
+
+struct PresentStyle {
+  static var numberFormatter: NumberFormatter {
+    let formatter = NumberFormatter()
+    formatter.numberStyle = .decimal
+    
+    return formatter
+  }
+  
+  static func formatNumber(_ number: Int) -> String {
+    guard let result = numberFormatter.string(for: number) else {
+      return ""
+    }
+    
+    return result
```

Comment:
이 내용은 자주 사용하는 형식이라 다음과 같은 문법으로도 제공하고 있어요. 
- [Nil-Coalescing Operator (Nil 병합 연산자) - Swift Programming Language 5.5](https://docs.swift.org/swift-book/LanguageGuide/BasicOperators.html#ID72)

```suggestion
    return numberFormatter.string(for: number) ?? ""
```